com.github.sdbg.debug.ui/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,9 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.ui.console,
  org.eclipse.debug.ui,
  com.github.sdbg.debug.core,
- org.eclipse.ltk.core.refactoring
+ org.eclipse.ltk.core.refactoring,
+ com.google.gdt.eclipse.core,
+ org.eclipse.jdt.core
```

Comment:
* Prio A: If I just apply the pull request, SDBG will become un-buildable, as it does not depend neither on the standard GPE, nor on the GPE version which has the IDebugLaunch interface. File com.github.sdbg.releng.targetplatform.target has to be changed to reflect this new dependency. I'll help you with that shortly...

* Prio A-B: The main core of SDBG is language independent. It does not know about JDT nor GWT and I think it is good to keep it that way. Therefore, let's move the DebugLaunch implementation, and the contribution to the extension point to the SDBG plugin called "com.github.sdbg.integration.jdt". This is the plugin dedicated for Java and GWT specific stuff.